app/views/games/show.html.erb

```diff
@@ -0,0 +1,5 @@
+<h1>Chessboard</h1>
+
+<table id='chessboard'>
```

Comment:
I have updated this pull request, I changed the JS code to Ruby and modified the games_controller.rb also 👍